feature-libs/product-configurator/rulebased/core/facade/configurator-place-order.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { OrderPlacedEvent } from '@spartacus/checkout/root';
+import { EventService } from '@spartacus/core';
+import { ConfiguratorActions } from '../state/actions/index';
+import { StateWithConfigurator } from '../state/configurator-state';
+
+@Injectable({ providedIn: 'root' })
+export class ConfiguratorPlaceOrderService {
+  constructor(
+    protected store: Store<StateWithConfigurator>,
+    protected eventService: EventService
+  ) {}
+
+  init(): void {
+    this.eventService.get(OrderPlacedEvent).subscribe(() => {
```

Comment:
Perhaps see if there would be a way to clean up the subscription to be on the safe side.

This is done with a Subscription object in some [other listener classes](https://github.com/SAP/spartacus/blob/develop/feature-libs/checkout/core/events/checkout-event.listener.ts#L83-L85)